pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -100,6 +100,7 @@ def __init__(
             checkpoint_io=checkpoint_io,
             precision_plugin=precision_plugin,
         )
+        log.verbose(f"Initializing DDP: {self.__class__.__name__}")
```

Comment:
this log line is possibly confusing with L226. 
```suggestion
        log.verbose(f"Initializing DDP Plugin: {self.__class__.__name__}")
```